src/algebra/covariant_and_contravariant.lean

```diff
@@ -51,7 +51,7 @@ However, sometimes as a **non-typeclass** assumption, we prefer `flip (*)` (or `
 as it is easier to use. -/
 
 -- TODO: convert `has_exists_mul_of_le`, `has_exists_add_of_le`?
--- TODO: relationship with `add_con`
+-- TODO: relationship with `con/add_con`
 -- include equivalence of `left_cancel_semigroup` with
 -- `semigroup partial_order contravariant_class α α (*) (≤)`?
 -- use ⇒, as per Eric's suggestion?
```

Comment:
"Eric's suggestion" is a little obscure; is there a link to a conversation on GitHub that could be added?

Reply:
Thanks, Bryan!  I just added a link to the relevant part of the Zulip suggestion: it was very brief and not followed up... at least yet!